### PR TITLE
SUS--1 | Small linter fixes

### PR DESCRIPTION
**Pull request by hakubo**
Started to use vscode
and it marks folders as red when there are problems in it

I don't like my folders red - so I fixed most of what it complains about

### CODE REVIEW

**Comment by macbre** [comment]
http://php.net/manual/en/language.basic-syntax.phptags.php says that

> PHP also allows for short open tag `<?` (which is discouraged since it is only available if enabled using the `short_open_tag` `php.ini` configuration file directive, or if PHP was configured with the` --enable-short`-tags option).

So, let's replace `<?` with `<?php`.

**Comment by mszabo-wikia** [comment]
yep, I'm not sure why the linter is complaining there since it is valid syntax

File: extensions/wikia/Wall/templates/WallHistory_index.php [hunk 1]
```diff
@@ -125,7 +125,7 @@
 			<?= $app->renderView( 'PaginationController', 'index', [ 'totalItems' => $totalItems, 'itemsPerPage' => $itemsPerPage, 'currentPage' => $currentPage, 'url' => $wallHistoryUrl ] ); ?>
 		<?php endif;?>
 	<?php endif; ?>
-<?php else: ?>
+<? else: ?>
```
Comment (macbre) [inline extensions/wikia/Wall/templates/WallHistory_index.php]: So this change (and the one below) is not needed, right?

Comment (hakubo) [reply extensions/wikia/Wall/templates/WallHistory_index.php]: grr... missed it :)

**Review by macbre — APPROVED** [review]

**Pull request merged**